hack/godep-save.sh

```diff
@@ -164,8 +163,13 @@ fork-with-fake-packages github.com/docker/docker \
 # kubernetes tests, we have to extract the missing test dependencies and run godep-save again
 # until we converge. Because we rsync kubernetes itself above, two iterations should be enough.
 MISSING_TEST_DEPS=""
+ALL_DIRS="$(os::util::list_go_src_files | cut -d '/' -f 1-2 | sort -u | grep -v "^./cmd"|grep -v doc.go)"
+ALL_DIRS+=" $(os::util::list_go_src_files | grep "^./cmd/"| cut -d '/' -f 1-3 | LC_ALL=C sort -u|grep -v doc.go)"
```

Comment:
This one is a bit different than the one in verify-govet.go as it also ignores doc.go (`grep -v doc.go`).